jujugui/static/gui/src/app/components/entity-content-readme/entity-content-readme.js

```diff
@@ -104,9 +104,9 @@ YUI.add('entity-content-readme', function() {
 
     render: function() {
       return (
-        <div className="entity-content_readme">
-          <h2 id="readme">Readme</h2>
-          <div className="entity-content_readme-content"
+        <div className="entity-content__readme">
+          <h2 className="entity-content__header" id="readme">Readme</h2>
```

Comment:
Those ids were there for linking to a particular section of a page with /apache2#readme etc. Our state system has a bit of a funky relationship with # links. urls such as `http://0.0.0.0:6543/?store=trusty/apache2#readme` will work but `http://0.0.0.0:6543/inspector/mysql/?store=trusty/apache2#readme` will not.

Not sure if that means we should leave them or remove them...